terraform/modules/aws-brig-prekey-lock-event-queue/README.md

```diff
@@ -0,0 +1,30 @@
+Terraform module: Brig pre-key locking & event queue 
```

Comment:
We need a single user, scoped to those permissions allowed on AWS.

Reply:
done with the last refactoring (putting the two modules back together). pls have another look.